### PR TITLE
frontend/feature/#1686/driver-wait-timer

### PR DESCRIPTION
**Pull request by mohdshoaib18169**
## Type of Change

- [ ] Bugfix
- [ x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
 Adding Wait Timer For Driver

ScreenShots -- 
1. https://drive.google.com/file/d/1SHaBnJ3J1XLuaxYLnSNqfreO9XEGGB9c/view?usp=sharing2. 
2. https://drive.google.com/file/d/1jDPf1jMVEerTdqBdca0YO660UbkbkzmH/view?usp=sharing3 3.https://drive.google.com/file/d/1ZxU72eNIGR5uSpQ6ljaQhFisMTOxOeOJ/view?usp=sharing

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by kavya-shree-s — CHANGES_REQUESTED** [review]

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs [hunk 1]
```diff
@@ -92,7 +92,12 @@ screen initialState =
               case (activeRideResponse.list DA.!! 0) of
                 Just ride -> lift $ lift $ doAff do liftEffect $ push $ RideActiveAction ride
                 Nothing -> setValueToLocalStore IS_RIDE_ACTIVE "false"
-
+          let startingTime = (HU.differenceBetweenTwoUTC (HU.getCurrentUTC "") (getValueToLocalStore SET_WAITING_TIME))
+          if ((getValueToLocalStore IS_WAIT_TIMER_STOP) == "Triggered") && initialState.props.timerRefresh  then do
```
Comment (aryan-13) [inline Frontend/ui-driver/src/Screens/HomeScreen/View.purs:96]: Can we `TRIGGERED` ,  `PostTriggered` and `Stop` as ENUM

File: Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs [hunk 2]
```diff
@@ -358,3 +360,35 @@ requestInfoCardConfig _ = let
...
+    }
+  , imageConfig {
+      imageUrl = "ny_ic_waiting_auto,https://assets.juspay.in/beckn/nammayatri/driver/images/ny_ic_ride_completed",
+      height = V 130,
+      width = V 130,
+      padding = Padding 0 0 6 0
+    }
+  , buttonConfig {
+      text = getString GOT_IT,
+      padding = Padding 0 16 0 20
```
Comment (aryan-13) [inline Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs:391]: Use `PaddingVertical` here

File: Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs [hunk 3]
```diff
@@ -358,3 +360,35 @@ requestInfoCardConfig _ = let
...
+  , primaryText {
+      text = getString HOW_LONG_WAITED_FOR_PICKUP,
+      fontSize = FontSize.a_16,
+      padding = Padding 16 16 0 0
+    }
+  , secondaryText {
+      text = getString CUSTOMER_WILL_PAY_FOR_EVERY_MINUTE,
+      visibility = VISIBLE,
+      fontStyle = FontStyle.regular LanguageStyle,
+      padding = Padding 16 0 0 0
```
Comment (aryan-13) [inline Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs:381]: Use `PaddingLeft` here

File: Frontend/ui-driver/src/Helpers/Utils.js [hunk 4]
```diff
@@ -321,6 +321,49 @@ export const setEnabled = function (id){
...
+            var callbackIOS = callbackMapper.map(function (timerId, sec) {
+              var minutes = getTwoDigitsNumber(Math.floor(sec / 60));
+              var seconds = getTwoDigitsNumber(sec - minutes * 60);
+              var timeInMinutesFormat = minutes + " : " + seconds;
+              cb(action(timerId)(timeInMinutesFormat)(sec))();
+            });
+            window.JBridge.startCountUpTimer(startingTime.toString(), callbackIOS);
+          }
+        } else {
+          console.log("hello1");
```
Comment (aryan-13) [inline Frontend/ui-driver/src/Helpers/Utils.js:344]: Remove this log

**Review by aryan-13 — CHANGES_REQUESTED** [review]

**Review by aryan-13 — APPROVED** [review]

**Review by Rohit4997 — CHANGES_REQUESTED** [review]

File: Frontend/android-native/app/src/driver/jatriSaathiDriver/res/drawable/ny_ic_waiting_auto.png
Comment (Rohit4997) [inline Frontend/android-native/app/src/driver/jatriSaathiDriver/res/drawable/ny_ic_waiting_auto.png:1]: Reduce size

Comment (mohdshoaib18169) [reply Frontend/android-native/app/src/driver/jatriSaathiDriver/res/drawable/ny_ic_waiting_auto.png:1]: reduced

**Review by Rohit4997 — APPROVED** [review]

**Pull request merged**